net-misc/ncman/ncman-0.0.1_p20190327-r3.ebuild

```diff
@@ -14,19 +17,15 @@ else
 	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
 fi
 
-DESCRIPTION="An ncurses UI for connman, forked from connman-json-client"
-HOMEPAGE="https://github.com/l4rzy/ncman"
-
-LICENSE="GPL-2"
+LICENSE="GPL-2+"
 SLOT="0"
-IUSE=""
 
 RDEPEND="dev-libs/json-c:0=
 	>=sys-apps/dbus-1.4
```

Comment:
```suggestion
RDEPEND="
	dev-libs/json-c:0=
	>=sys-apps/dbus-1.4
```

Reply:
Right. I'm looking at new ebuilds with my kde proj trained eye, and usually we do put lists of more-than-one dependencies into a neatly aligned block. @benkohler: what do you think?

Reply:
> Right. I'm looking at new ebuilds with my kde proj trained eye, and usually we do put lists of more-than-one dependencies into a neatly aligned block. @benkohler: what do you think?

Sure, let's make that change.  I believe the current formatting was just inherited from the ebuild of its predecessor connman-json-client which was inherited by myself years ago.